src/java.com/homework6/com/flower/kyiv/Flower.java

```diff
@@ -0,0 +1,12 @@
+package homework6.com.flower.kyiv;
```

Comment:
The same
package homework6.com.flower.kyiv; -> package com.flower.kyiv.entity; or package com.flower.kyiv.domain;
Choose the same way